jfrog_platform_self-hosted/assets/dashboards/jfrog_xray_logs_(self-hosted).json

```
@@ -0,0 +1,248 @@
+{
+    "author_name": "Datadog",
+    "description": "This dashboard provides a summary of access, service, and traffic log volumes associated with Xray. Additionally, customers are also able to track various HTTP response codes, HTTP 500 errors, and log errors for greater operational insight\n\nThe source code is also available on [Github](https://github.com/jfrog/log-analytics-datadog) ",
+    "layout_type": "ordered",
+    "template_variables": [],
+    "title": "JFrog Xray Logs (Self-hosted)",
+    "widgets": [
+        {
+            "definition": {
+                "has_background": true,
+                "has_border": true,
+                "horizontal_align": "center",
+                "sizing": "contain",
+                "type": "image",
+                "url": "https://raw.githubusercontent.com/jfrogtraining/integrations-extras/master/jfrog_platform/assets/images/xray.png",
```

Can you replace these URLs with:

```suggestion
                "url": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/assets/images/xray.png",
```

This link won't work until this PR is merged, but any github links should point to the Datadog org copy instead of the fork (it looks like there's a couple cases of this)